WalletWasabi.Fluent/Helpers/TransactionFeeHelper.cs

```diff
@@ -1,12 +1,15 @@
 using System.Collections.Generic;
 using System.Linq;
 using NBitcoin;
+using WalletWasabi.Fluent.ViewModels.Wallets.Send;
 using WalletWasabi.Wallets;
 
 namespace WalletWasabi.Fluent.Helpers;
 
 public static class TransactionFeeHelper
 {
+	public const decimal FeePercentageThreshold = 130;
```

Comment:
In #7028 @kiminuo used 1.25 (125) as a `Max Extra Fee`, shouldn't use the same values?

https://github.com/zkSNACKs/WalletWasabi/blob/5e4f6f791f033b23c7c1ad31d41a20f873bbc84a/WalletWasabi/Blockchain/TransactionBuilding/ChangelessTransactionCoinSelector.cs#L14-L15

Reply:
`MaxExtraFee` is about the payment fee.
`FeePercentageThreshold ` is about the transaction (mining) fee.
The meaning is different.

Mine comes from here https://github.com/zkSNACKs/WalletWasabi/issues/6465#issuecomment-938838209.
But it is just guessed number, if you feel like feel free to modify it to `125` for the sake of consistency.

Reply:
> MaxExtraFee is about the payment fee.

I opened #7099 to fix the name of that variable because it is about the payment and not the fee.